docs/libcurl/symbols-in-versions

```diff
@@ -1146,6 +1146,7 @@ CURLWARNING                     7.66.0
 CURLWS_BINARY                   7.86.0
 CURLWS_CLOSE                    7.86.0
 CURLWS_CONT                     7.86.0
+CURLWS_FIN                      8.13.0
```

Comment:
Wasn't sure what to do here, but I needed this symbol represented in this file for the test to pass